demo/conf/config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
 	app: {
 		API_PATH: 'api',
```

Comment:
@petrKavulok Can you also please remove this line, which is kinda obsolete? Thanks